### PR TITLE
Add simple launcher for running rust application

### PR DESCRIPTION
**Pull request by xilec**
This launcher just run compiled exe-file without any arguments and without debug, but any way it much more comfortable than run from console by hand.

### CODE REVIEW

File: VisualRust.Project/DefaultRustLauncher.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        }
+
+        private ProcessStartInfo CreateProcessStartInfoNoDebug(string startupFile)
+        {
+            // TODO add command line arguments
+            var commandLineArgs = string.Empty;
+            var startInfo = new ProcessStartInfo(startupFile, commandLineArgs);
+
+            startInfo.UseShellExecute = false;
+            startInfo.WorkingDirectory = _project.GetWorkingDirectory();
```
Comment (vosen) [inline VisualRust.Project/DefaultRustLauncher.cs]: This line should be removed.
C# projects launch their executables with default working directory (usually <solution>/<project>/<bin>/<Configuration>), we should do the same.

File: VisualRust.Project/DefaultRustLauncher.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+            return startInfo;
+        }
+
+        private void StartWithDebugger(string filePath)
+        {
+            throw new NotImplementedException();
+        }
```
Comment (vosen) [inline VisualRust.Project/DefaultRustLauncher.cs]: Remove `StartWithDebugger(...)` and simply execute `StartWithoutDebugger(...)` in `LaunchFile(...)` without checking for debug flag.
First thing that potential user is going to do is to press big green "Start" arrow. It's better to launch without debugging than to give him "Not supported" message box.

**Comment by vosen** [comment]
Very nice, make those two changes I noted and I'm merging this PR.

**Comment by xilec** [comment]
Needed changes is done.

**Comment by vosen** [comment]
Good job!

**Pull request merged**